### PR TITLE
[JetLagged] Add drawer with gesture animation 

### PR DESCRIPTION
**Pull request by riggaroo**
https://github.com/android/compose-samples/assets/9973046/6b6824fb-ed59-45b6-8221-8dcc1b0bcc54

### CODE REVIEW

**Review by OkuiohSd — COMMENTED** [review]
My mony

File: JetLagged/app/src/main/java/com/example/jetlagged/JetLaggedDrawer.kt [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+                Surface(
+                    color = Color.White,
+                    modifier = Modifier.fillMaxSize()
+                ) {
+                }
+        }
+    }
+}
+
+enum class DrawerState {
```
Comment (bentrengrove) [inline JetLagged/app/src/main/java/com/example/jetlagged/JetLaggedDrawer.kt:199]: nit: Could probably be private?

File: JetLagged/app/src/main/java/com/example/jetlagged/JetLaggedDrawer.kt [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.util.lerp
+import kotlinx.coroutines.launch
+
+@Composable
+fun HomeScreenDrawer() {
+    Surface(
+        modifier = Modifier.fillMaxSize(),
+        color = Color.White
```
Comment (bentrengrove) [inline JetLagged/app/src/main/java/com/example/jetlagged/JetLaggedDrawer.kt:60]: nit: Should this be using a themed value rather than hardcoding to white?

**Review by bentrengrove — APPROVED** [review]
LGTM!

Very minor comments, feel free to ignore.

File: JetLagged/app/src/main/java/com/example/jetlagged/JetLaggedDrawer.kt (hunk 2 above)
Comment (riggaroo) [reply JetLagged/app/src/main/java/com/example/jetlagged/JetLaggedDrawer.kt]: fixed

File: JetLagged/app/src/main/java/com/example/jetlagged/JetLaggedDrawer.kt (hunk 1 above)
Comment (riggaroo) [reply JetLagged/app/src/main/java/com/example/jetlagged/JetLaggedDrawer.kt]: fixed

**Pull request merged**